pkg/transforms/filter.go

```diff
@@ -46,117 +48,67 @@ func NewFilterOut(filterValues []string) Filter {
 // If FilterOut is false, it filters out those Events not associated with the specified Device Profile listed in FilterValues.
 // If FilterOut is true, it out those Events that are associated with the specified Device Profile listed in FilterValues.
 func (f Filter) FilterByProfileName(edgexcontext *appcontext.Context, params ...interface{}) (continuePipeline bool, result interface{}) {
-	mode := "For"
-	if f.FilterOut {
-		mode = "Out"
+	event, err := f.setupForFiltering("FilterByProfileName", "ProfileName", edgexcontext.LoggingClient, params...)
```

Comment:
define a constant for "ProfileName" if shared by others

Reply:
No shared, one time use.